src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -226,6 +226,61 @@ public class XdocsPagesTest {
     private static final Set<String> GOOGLE_MODULES = Collections.unmodifiableSet(
         CheckUtil.getConfigGoogleStyleModules());
 
+    // Contains all the sections which are not migrated to chapter wise testing
+    // in google style documentation.
+    // This list will be removed once all the sections are migrated.
```

Comment:
please put here `until #xxxxxx`
to make sure in scope of what ticket we will do final cleanup.

Reply:
Currently, we does not have any issue which talks about migrating to chapter wise testing, so which issue should I link here?

Reply:
lets create such issue and reference it here :)

Reply:
I have opened the issue: https://github.com/checkstyle/checkstyle/issues/14937. I'll link this as you requested.

Reply:
Linked the issue: https://github.com/checkstyle/checkstyle/blob/7e868f5cd0424fa926ed50a7426158666c3941a1/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L229-L233